source/_patterns/basics/dropdown/02-dropdown-states.mustache

```diff
@@ -0,0 +1,8 @@
+{{> basics-dropdown(dropdownText: "Closed Dropdown") }}
+{{> basics-dropdown(dropdownText: "Open Dropdown", dropdownState: "open") }}
+<br>
```

Comment:
Dropdowns need to be triggered manually in bs4, so I put these here so people could see both versions. Is there a way to add JS to a basic? If so, I'll do it that way instead